README.md

```diff
@@ -29,3 +29,17 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+The main loop of my code runs once for each element of the list, as n is set to equal the length of the list. For each element of the list, flip is called twice. Once to get the biggest pancake to the top of the stack, and then once more to get that pancake on top of the already sorted list.
+So the runtime ($\Theta$) for this algorithm is ($\Theta$) $(n^2)$.
```

Comment:
What is the runtime in terms of the number of comparisons and the number of flips?

Reply:
I provided runtime in terms of flips. I'm not sure what the runtime would be for in terms of comparisons, as far as I can tell I never compare the list, unless that is asking for when I find the max value of each sublist? Can you please clarify for me?

Reply:
How often are you comparing elements in the list? And the number of flips isn't quadratic.

Reply:
Number of flips is 2n, I don't know why I went quadratic. I'm comparing every element in each sublist, so thats n then n-1 and so on. I'm not sure how I would write that asymptotically.

Reply:
Sounds similar to e.g. the worst case of quicksort.